src/transformers/models/trocr/processing_trocr.py

```diff
@@ -61,7 +69,15 @@ def __init__(self, image_processor=None, tokenizer=None, **kwargs):
         self.current_processor = self.image_processor
         self._in_target_context_manager = False
 
-    def __call__(self, *args, **kwargs):
+    def __call__(
+        self,
+        images: ImageInput = None,
+        *,  # TODO deprecate/remove this
```

Comment:
Up to you, there is no deprecation cycle so this is not very actionnable, neither for us nor for the user!

Reply:
Removed the `*`.
cc @molbap @qubvel